### PR TITLE
Solidification spec: Follow standard layout

### PR DESCRIPTION
**Pull request by jkrvivian**

### CODE REVIEW

File: 4.4 Solidification.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # 4.4 Solidification
 A message is solid if all its parents are stored, solid and valid. This section defines how messages get solid in the Tangle.
 
+The Solidification specification depends on the following specifications:
+- 4.2 - Timestamps
```
Comment (darcy-camargo) [inline 4.4 Solidification.md:5]:
```suggestion
- [4.2 - Timestamps](./4.2%20Timestamps.md)
```

File: 4.4 Solidification.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+- 5.1 - UTXO
```
Comment (darcy-camargo) [inline 4.4 Solidification.md:6]:
```suggestion
- [5.1 - UTXO](./5.1%20UTXO.md)
```

File: 4.4 Solidification.md [hunk 3]
```diff
@@ -9,9 +13,8 @@ In that way, the Tangle enables all nodes to retrieve all of a message's history
 ## 4.4.2 Definitions
 * **valid**: A message is considered valid if it passes the following filters from the solidifier and from the message booker:
     * solidifier: it checks if parents are valid,
-    * booker: it check if the contained transaction is valid. Notice that only messages containing a transaction are required to perform this check..
-> TODO: link to message layout semantic check, and transaction syntactic/semantic check in ledgerstate spec.
-* **parents age check**: A check that ensures the timestamps of parents and child are valid, following the details defined in [Section 4.2 - Timestamps](./4.2%20Timestamps.md).
+    * booker: it check if the contained transaction is valid. Notice that only messages containing a transaction are required to perform this check, which are defined in [5.1 - UTXO](./5.1%20UTXO.md#validation)
+* **parents age check**: A check that ensures the timestamps of parents and child are valid, following the details defined in [4.2 - Timestamps](./4.2%20Timestamps.md).
```
Comment (darcy-camargo) [inline 4.4 Solidification.md:17]:
```suggestion
* **parents age check**: A check that ensures the timestamps of parents and child are valid, following the details defined in [Section 4.2 - Timestamps](./4.2%20Timestamps.md).
```

File: 4.4 Solidification.md [hunk 4]
```diff
@@ -23,11 +26,11 @@ During solidification, if a node is missing a referenced message, the correspond
 
 If a message gets solid, it shall walk through the rest of the data flow, then propagate the solid status its future cone by performing the solidification checks on each of the message in its future cone again.
 
-Figure 4.4.4.1 shows the solidification process:
+Figure 4.4.1 shows the solidification process:
 
 ![GoShimmer-flow-solidification_spec](https://user-images.githubusercontent.com/11289354/117009286-28333200-ad1e-11eb-8d0d-186c8d8ce373.png)
 <p style="text-align: center;">
-    Figure 4.4.4.1, solidification workflow
+    Figure 4.4.1, solidification workflow
```
Comment (oliviasaa) [inline 4.4 Solidification.md:33]:
```suggestion
    Figure 4.4.1: solidification workflow
```

File: 4.4 Solidification.md [hunk 5]
```diff
@@ -55,6 +58,9 @@ Below we define the form of SolidificationRequest and SolidificationResponse:
          <td>Contains the message ID of the requested message.</td>
      </tr>
  </table>
+<p style="text-align: center;">
+    Table 4.4.2, SolidificationRequest data fields description
```
Comment (oliviasaa) [inline 4.4 Solidification.md:62]:
```suggestion
    Table 4.4.2: SolidificationRequest data fields description
```

File: 4.4 Solidification.md [hunk 6]
```diff
@@ -74,7 +80,9 @@ Below we define the form of SolidificationRequest and SolidificationResponse:
          <td>Contains the entire requested message.</td>
      </tr>
  </table>
-
+<p style="text-align: center;">
+    Table 4.4.3, SolidificationResponses data fields description
```
Comment (oliviasaa) [inline 4.4 Solidification.md:84]:
```suggestion
    Table 4.4.3: SolidificationResponses data fields description
```

**Review by oliviasaa — APPROVED** [review]
I left some suggestions (since we are using ":" instead of "," )

**Pull request merged**